src/topp/FileFilter.cpp

```diff
@@ -135,6 +135,69 @@ class TOPPFileFilter :
   }
 
 private:
+  static FeatureMap makeUnique(const FeatureMap& feature_map) 
```

Comment:
Could potentially be a template to be used with feature map and consensus map.
Consider using BaseFeature* as this is exists for both feature and consensus feature

Reply:
We thought so as well. We recently removed a lot of templates, but guess it makes sense here.